cico_functions.sh

```diff
@@ -33,6 +33,32 @@ function load_jenkins_vars() {
   fi
 }
 
+# Sorts two versions and returns the lower version to check if its supported for using docker buildx.
+function check_supported_version() {
+  local query=$1
+  local target=$2
+  echo "$target" "$query" | tr ' ' '\n' | sort -V | head -n1 2> /dev/null
+}
+
+# Checks whether docker and kernel versions are greater than or equal to the specified version, such that docker buildx requirements are met.
+function check_buildx_support() {
+  docker_version="$(docker --version | cut -d' ' -f3 | tr -cd '0-9.')"
+  if [[ $(check_supported_version "$docker_version" "19.03") != 19.03 ]]; then
+    echo "CICO: Docker $docker_version greater than or equal to 19.03 is required."
```

Comment:
I just suspect that centos CI has the version lower than 19.03 - we might need to install new version of docker manually